spec/request/user_request_spec.rb

```diff
@@ -159,21 +159,23 @@
           .and not_trigger_statsd_increment('api.external_http_request.MVI.skipped')
           .and not_trigger_statsd_increment('api.external_http_request.MVI.success')
       end
+      expect(MVI::Configuration.instance.breakers_service.latest_outage.start_time.to_i).to eq(start_time.to_i)
```

Comment:
We could potentially test that another service on the same hostname didn't have an outage started at the same time as an integration test on the matcher.